includes/shareable-content.php

```diff
@@ -88,7 +88,7 @@
  */
 $pixel = sprintf(
 	// %1$s is the javascript source, %2$s is the post ID, %3$s is the plugins URL
-	'<img id="republication-tracker-tool-source" src="%1$s/?republication-pixel=true&post=%2$s&ga=%3$s" style="max-width:200px;">',
+	'<img id="republication-tracker-tool-source" src="%1$s/?republication-pixel=true&post=%2$s&ga=%3$s" style="max-width:1em;">',
```

Comment:
Should this be max-width or max-height?

Reply:
I think either way makes sense.

Reply:
If the image we're grabbing is square, that's true.